### ISSUE TITLE
Enable ability to manually configure which blog items are shown in blog list page and in the left side panel 

### ISSUE THREAD

**Issue by pranay01** [SigNoz/signoz.io#102]
Currently the blog list page shows blogs based on recency. We should have flexibility to also configure this list based on what we want to highlight. 

The left side panel in any blog automatically shows recent articles. We should also have ability to manually configure it to show blogs related to the current blog

**Comment by zriyansh** [comment]
Also filter/ sort blogs according to date and number of blogs in each tag. 

The panel can also show just the title of each blog present so users can make a better decision on which all they want to read.

A workaround can be to manually list each blog (Title and 1 line summary) on a separate created page in docs and mention this all. Won't take much time, just need to scrap data out and format it currently in markdown format.

**Comment by Calm-Rock** [comment]
@pranay01 is this issue still relevant?

**Comment by pranay01** [comment]
Yes, It still is. More related to website structure though - will add a `website` tag
